be/src/formats/parquet/encoding_plain.h

```diff
@@ -175,6 +180,76 @@ class PlainDecoder<Slice> final : public Decoder {
         return Status::OK();
     }
 
+    Status next_batch_with_nulls(size_t count, const NullInfos& null_infos, ColumnContentType content_type, Column* dst,
+                                 const FilterData* filter) override {
+        const uint8_t* __restrict is_nulls = null_infos.nulls_data();
+        // fill null data
+        CHECK(dst->is_nullable());
+        size_t null_cnt = null_infos.num_nulls;
+        if (dst->is_nullable()) {
```

Comment:
Is this `if nullable check` necessary? there is a `CHECK nullable` before.

Reply:
changed to DCHECK